lib/screens/home/explorer/staking/validator_list_item.dart

```diff
@@ -25,16 +26,21 @@ class ValidatorListItem extends StatelessWidget {
         }
         final bloc = get<StakingFlowBloc>();
         final stakingDetails = bloc.stakingDetails.value;
+
         Delegation? delegation;
+        Rewards? rewards;
         try {
           delegation = stakingDetails.delegates
               .firstWhere((element) => element.sourceAddress == item.addressId);
+          rewards = bloc.stakingDetails.value.rewards.firstWhere(
+              (element) => element.validatorAddress == item.addressId);
```

Comment:
Perhaps use `firstWhereOrNull` from `dart:collection` instead of `try/finally`

Reply:
I wasn't aware that actually existed. Cool.

Reply:
Committed these changes.